# Toronto Clustering and Segmenting


### Part I: Clean Dataframe and Determine Shape

In [22]:
# The code was removed by Watson Studio for sharing.

Postal Code     Borough                             Neighborhood
0         M3A  North York                                Parkwoods
1         M6A  North York         Lawrence Manor, Lawrence Heights
2         M9A   Etobicoke  Islington Avenue, Humber Valley Village
3         M3B  North York                                Don Mills
4         M6B  North York                                Glencairn

In [23]:
# Determine shape of dataframe before removing the Boroughs that have Not assigned

df_data_1.shape

(180, 3)

In [25]:
# Delete rows where the Borough is Not assigned

df_tor = df_data_1[df_data_1.Borough != 'Not assigned']
df_tor.head(10)

Postal Code     Borough                                       Neighborhood
0          M3A  North York                                          Parkwoods
1          M6A  North York                   Lawrence Manor, Lawrence Heights
2          M9A   Etobicoke            Islington Avenue, Humber Valley Village
3          M3B  North York                                          Don Mills
4          M6B  North York                                          Glencairn
5          M9B   Etobicoke  West Deane Park, Princess Gardens, Martin Grov...
6          M3C  North York                                          Don Mills
7          M6C        York                                 Humewood-Cedarvale
8          M9C   Etobicoke  Eringate, Bloordale Gardens, Old Burnhamthorpe...
10         M6E        York                                Caledonia-Fairbanks

In [26]:
# Determine shape of updated dataframe

df_tor.shape

(103, 3)

The number or rows was reduced from 180 to 103 after the Boroughs with no assignments were removed.